app/utils/tables.py

```diff
@@ -10,10 +10,10 @@
 
 
 def _format_currency(number: Decimal | None, currency: str) -> str:
-    if isinstance(number, str):
+    if isinstance(number, str) or not number:
```

Comment:
Why not 

```
if number is None:
    return "-"
```

Seems odd to call int on values like None, “” and 0.

Reply:
I did that first but wasn't sure what was better, an additional if/return or the current approach. I will change it to the additional if then

Reply:
Clarity over brevity :)